STP-3470: Clarify SAT 2.5 and IUF relationship in integration (#86)
Cray-HPE/docs-sat: docs/usage/sat_bootprep.md

# SAT Bootprep

SAT provides an automated solution for creating CFS configurations, building
and configuring images in IMS, and creating BOS session templates. The
solution is based on a given input file that defines how those configurations,
images, and session templates should be created. This automated process centers
around the `sat bootprep` command. Man page documentation for `sat bootprep`
can be viewed similar to other SAT commands.

```screen
ncn-m001# sat-man sat-bootprep
```

> The `sat bootprep` command helps the Install and Upgrade Framework (IUF) install,
> upgrade, and deploy products on systems managed by CSM. Outside of IUF, you will
> rarely need to use `sat bootprep`. For more information on this relationship,
> see [SAT and IUF](sat_and_iuf.md). For more information on IUF, see the
> [IUF section](https://cray-hpe.github.io/docs-csm/en-14/operations/iuf/iuf/) of
> the [Cray System Management Documentation](https://cray-hpe.github.io/docs-csm/).

Reviewer: Suggest weakening "rare" to "uncommon" and possibly using the section name in the link to the IUF docs in docs-csm:

```suggestion
The `sat bootprep` command helps the Install and Upgrade Framework (IUF)
install, upgrade, and deploy products on systems managed by CSM. It is uncommon
to use `sat bootprep` outside the context of the IUF. For more information on
this relationship, see [SAT and IUF](sat_and_iuf.md). For more information on
IUF, see [Install and Upgrade Framework](https://cray-hpe.github.io/docs-csm/en-14/operations/iuf/iuf/)
in the [Cray System Management Documentation](https://cray-hpe.github.io/docs-csm/).
```

Author: Updating in next commit similar to what we agreed for these comments in the SAT_and_IUF.md file.

  under the `session_templates` key. The value of this variable is the name of
  the IMS image used in this session template.

>   **Note:** The name of a session template is restricted to 45 characters. Keep
>   this in mind when creating a variable with `image.name`.

Reviewer: ```suggestion
  **Note:** The name of a session template is restricted to 45 characters. Keep
  this in mind when using `image.name` in the name of a session template.
```

Author: Updated in next commit.
